sovtokenfees/sovtokenfees/test/dynamic_validation/test_nym_cannot_pay.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import pytest
+from sovtokenfees.test.helper import add_fees_request_with_address, ensure_all_nodes_have_same_data, \
+    get_amount_from_token_txn
+
+from plenum.common.exceptions import RequestRejectedException
+
+from plenum.test.helper import sdk_send_and_check, sdk_sign_request_from_dict
+
+from indy_common.authorize.auth_map import auth_map, add_new_identity_owner
+
+from indy_common.constants import NYM
+
+from indy_node.test.auth_rule.helper import sdk_send_and_check_auth_rule_request
+
+from indy_common.authorize.auth_actions import ADD_PREFIX
+
+
+def test_validation_nym_with_fees_cannot_pay(fees,
```

Comment:
I would split the test into five:
1) Send Nym with FEEs when Fees are not required (expect Reject)
2) Send Nym without FEEs when Fees are required (expect Reject)
3) Send Nym without less FEEs than required (expect Reject)
4) Send Nym with more FEEs than required (expect Reject)
5) Send Nym with correct amount of Fees